app/decorators/corporate_decorator.rb

```diff
@@ -0,0 +1,25 @@
+class CorporateDecorator < ContentItemDecorator
+  attr_reader :extra_content_path
+
+  def initialize(object, options = {})
```

Comment:
You could reduce this signature down to `initialize(*)` and this would be passed along to `super`.